src/calculation/MeasureReportBuilder.ts

```diff
@@ -550,8 +671,8 @@ export default class MeasureReportBuilder<T extends PopulationGroupResult> {
     });
   }
 
-  public getReport(): fhir4.MeasureReport {
-    this.calculateGroupScores();
+  public getReport(results?: ExecutionResult<PopulationGroupResult>[]): fhir4.MeasureReport {
```

Comment:
The method signature for this should not have been changed. This is intended to be called after all patients (or single) have had their results added with `addPatientResults`. This function should just be completing the MeasureReport and returning it, not taking any additional data.

Reply:
Ah that makes sense - this was the only way that I could come up with for getting all the data needed for computing an aggregate measure score but I agree that it should not be changed. Did we confirm that the "count" for each measure observation should be the observation value rather than the count of how many measure observations there are? Under that assumption, I think some refactoring could be done that would make it easier to access all the necessary data for calculating the aggregate measure score. If not, do you know how to get all the observations for all patients in one place for calculating the aggregate measure score that does not require bringing in the entire `ExecutionResult`?

Reply:
I think for this specific measure it will be the sum of all measures observations across all patients. So I think you could add a field to the MeasureReportBuilder instance that is used to hold on to or progressively count all of the observations for the numerator and denominator as each patient result is added. Then when getReport is called it can use that to make the overall score from that instance data instead of using the ExecutionResults.